src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -36,13 +36,13 @@ public override IEnumerable<string> PreprocessorSymbolNames
         }
 
         public CSharpParseOptions(
-            LanguageVersion languageVersion = LanguageVersion.CSharp7,
+            LanguageVersion languageVersion = LanguageVersion.Latest,
             DocumentationMode documentationMode = DocumentationMode.Parse,
             SourceCodeKind kind = SourceCodeKind.Regular,
             IEnumerable<string> preprocessorSymbols = null)
-            : this(languageVersion, documentationMode, kind, preprocessorSymbols.ToImmutableArrayOrEmpty())
+            : this(languageVersion.MapLatestToVersion(), documentationMode, kind, preprocessorSymbols.ToImmutableArrayOrEmpty())
         {
-            if (!languageVersion.IsValid())
+            if (!LanguageVersion.IsValid())
```

Comment:
I will add a comment. Briefly, the parameter is mapped when passed to the this(...) constructor. We want to be
testing the mapped value, not the parameter's unmapped value.

---
In reply to: [72671474](https://github.com/dotnet/roslyn/pull/12796#discussion_r72671474) [](ancestors = 72671474)